server/repository/src/migrations/v1_06_00/mod.rs

```diff
@@ -19,6 +21,8 @@ impl Migration for V1_06_00 {
         plugin_data::migrate(connection)?;
         master_list::migrate(connection)?;
         temperature_breach::migrate(connection)?;
+        item_link_create_table::migrate(connection)?;
+        item_line_add_item_link_id::migrate(connection)?;
```

Comment:
nit: I know mSupply names it this... but can this be renamed to stock_line_... 😄